Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ContextFactory.cs

```diff
@@ -1,40 +1,37 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Data.Common;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Fakebook.Profile.DataAccess;
+using Fakebook.Profile.DataAccess.EntityModel;
 using Microsoft.Data.Sqlite;
 using Microsoft.EntityFrameworkCore;
 
-namespace Fakebook.Profile.UnitTests.APITests
+namespace Fakebook.Profile.UnitTests.ApiTests
```

Comment:
the folder name is still APITests

Reply:
Not sure how to fix this.. I renamed the folder locally but git doesn't seem to track that change.

Reply:
@tdunbar123 https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git

Reply:
Renamed folder from APITests to ApiTests